chemprop/cli/predict.py

```diff
@@ -283,14 +291,34 @@ def make_prediction_for_model(
             f"pred_{i}" for i in range(preds.shape[1])
         ]  # TODO: need to improve this for cases like multi-task MVE and multi-task multiclass
 
-    df_test[target_columns] = preds
+    df_test = pd.read_csv(args.test_path)
+    df_test[target_columns] = average_preds
     if output_path.suffix == ".pkl":
         df_test = df_test.reset_index(drop=True)
         df_test.to_pickle(output_path)
     else:
         df_test.to_csv(output_path, index=False)
     logger.info(f"Predictions saved to '{output_path}'")
 
+    if len(model_paths) > 1:
+        individual_preds = torch.concat(individual_preds, 1)
+        target_columns = [
+            f"{col}_model_{i}" for i in range(len(model_paths)) for col in target_columns
```

Comment:
Maybe we should add a `logger.info` that prints the mapping of `model_{i}` to the list of model_paths given. I could see a user being confused if they give `--model-path model1.pt model3.pt` and then see predictions for model_0 and model_1.

Reply:
Thanks! That's a great suggestion, and I'll add it.